docs/all_development/contributing/dev_docs.rst

```diff
@@ -0,0 +1,20 @@
+.. Some information copied from edx-documentation/en-us/developers/read_me.rst
+
+.. _Contributing to Open edX developer documentation:
+
+#################################################
+Contributing to Open edX developer documentation
+#################################################
+
+The **edX Developer Documentation** is based on `RST`_ source files, and `Sphinx`_ uses the source files to create the finished docs.
+
+You, as part of the user community, can help update and revise this documentation project
+on GitHub.
+
+https://github.com/edx/edx-developer-docs
```

Comment:
What is this URL for?

Reply:
It's the location of the developer docs repository ... the significance should be clear from the context. @robrap  what say you?